src/plugins/recordTypes/repatriationclaim/fields.js

```diff
@@ -615,11 +621,11 @@ export default (configContext) => {
               [config]: {
                 messages: defineMessages({
                   fullName: {
-                    id: 'field.nagpraclaims_common.status.fullName',
+                    id: 'field.repatriationclaims_common.status.fullName',
                     defaultMessage: 'Claim status state',
                   },
                   name: {
-                    id: 'field.nagpraclaims_common.status.name',
+                    id: 'field.repatriationclaims_common.status.name',
                     defaultMessage: 'State',
```

Comment:
One thing I realized I left out was updating the message for the `status` field. Since the group message is `Claim status` I originally went with `Claim status state`, which Kristina mentioned feels awkward compared to the `Claim documentation status` below. I'm not sure if just `Claim status`/`Status` would be ok or if we should switch to `Value`. Just a minor thing and I believe it's the last thing to sort out.

Reply:
Maybe Description/Claim status description?

Reply:
I ended up going with `Value`; we'll see how it goes. It feels a little awkward but feels ok when referring to the full name of `claim status value`.